### DISCUSSION TITLE
Pavlovia Error Uploading Data After Working For Months?

### DISCUSSION THREAD

**Discussion by evanknep** [jspsych/jsPsych#3247] [Q&A]
Hi all,

I've been running an online task using jspsych that is hosted on pavlovia. Now after completing the task, I get the error screen below. How do I go about troubleshooting this? 
<img width="635" alt="Screenshot 2024-03-08 at 2 13 29 PM" src="https://github.com/jspsych/jsPsych/assets/63071368/14dd18e2-811a-438a-9fe0-8778cc368a2e">

Thanks

**Comment by jodeleeuw** [comment]
Hi @evanknep,

I recommend checking with the Pavlovia team. They created the plugin and are probably the most likely to be able to help.
